site/website/pages/en/demos.js

```diff
@@ -2,60 +2,20 @@ const React = require('react');
 const CompLibrary = require('../../core/CompLibrary');
 const Container = CompLibrary.Container;
 const MarkdownBlock = CompLibrary.MarkdownBlock;
+const cardStyle = require(process.cwd() + "/static/jss/card.js");
```

Comment:
shouldn't you use relative path here?

Reply:
When I tried to use a relative path, I got an error for every path I tried. I think the docusaurus does not support it. When I looked at how to import this jss component, I came across [this](https://docusaurus.io/docs/en/api-pages#page-require-paths) and applied it.